cmd/edit/cmd.go

```diff
@@ -44,7 +44,7 @@ func init() {
 	Cmd.AddCommand(addon.Cmd)
 	Cmd.AddCommand(cluster.Cmd)
 	Cmd.AddCommand(ingress.Cmd)
-	Cmd.AddCommand(machinepool.Cmd)
+	machinepoolCommand := machinepool.NewEditMachinePoolCommand()
```

Comment:
Variables should be declared/initialised as close as possible to where they are used. Please move this to line 59.